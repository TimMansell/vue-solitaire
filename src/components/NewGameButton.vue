<template>
  <Button
    :is-stacked="true"
    :check-disabled="true"
    to-page="new"
    data-test="new-game-btn"
  >
    New
  </Button>
</template>

<script>
import Button from './Button.vue';

export default {
  name: 'NewGameButton',
  components: {
    Button,
  },
};
</script>
