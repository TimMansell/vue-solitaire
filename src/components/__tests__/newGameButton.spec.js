import { shallowMount } from '@vue/test-utils';
import NewGameButton from '@/components/NewGameButton.vue';

describe('NewGameButton.vue', () => {
  it('matches snapshot', () => {
    const wrapper = shallowMount(NewGameButton);

    expect(wrapper).toMatchSnapshot();
  });
});
