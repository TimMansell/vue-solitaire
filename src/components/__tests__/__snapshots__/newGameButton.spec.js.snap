// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NewGameButton.vue matches snapshot 1`] = `
<button-stub type="default" isstacked="true" checkdisabled="true" size="md" topage="new" data-test="new-game-btn">
  New
</button-stub>
`;
